clang/test/Sema/precision-and-size-conversion.c

```diff
@@ -0,0 +1,50 @@
+// RUN: %clang_cc1 -fsyntax-only -verify -Wimplicit-float-conversion -Wdouble-promotion -Wimplicit-float-size-conversion \
```

Comment:
Can we use -verify="abc" as done here  https://github.com/intel/llvm/commit/27305a0c9a47cf980e7de71d1d2548bb2400ea03 instead of using predefines?